src/index.html

```diff
@@ -17,6 +30,38 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <img
+        src="images/logo.png"
+        alt="logo"
+        class="image"
+      />
```

Comment:
The logo image should be wrapped in an anchor tag to meet the task requirements. This will make the logo clickable and usually links back to the homepage.